src/components/atoms/Input/index.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { FC } from "react";
+
+interface IInput {
+  type: string;
+  name: string;
+  value?: string | number;
+  ref?: React.Ref<HTMLInputElement>;
+  placeholder?: string;
+  onChange?: (event: React.ChangeEvent<HTMLInputElement>) => void;
+  variant?: string;
+}
+
+const Input: FC<IInput> = ({
+  type,
+  name,
+  value,
+  ref,
+  placeholder,
+  onChange,
+  variant,
+}) => {
+  const searchInputStyle: string = "w-full h-full bg-transparent";
+  const hiddenInputStyle: string = "hidden";
+  const defaultInputStyle: string =
+    "w-full h-[50px] p-4 bg-neutral-850 border-b-2 rounded border-orange-550";
+
+  const inputVariant =
```

Comment:
ternary operator use only for 2 scenarios (if u are using for more it is bad practice)